Backend_Repo/website/template/website/base.html

```diff
@@ -16,13 +16,23 @@
 	<link rel="stylesheet" href= "{% static 'css/materialize.min.css' %}">
 	<script type="text/javascript" src="{% static 'js/jquery-2.1.1.min.js' %}"></script>
 	<script src="{% static 'js/materialize.min.js' %}"></script>
-
+    <style>
```

Comment:
Do we want to maintain same styling for buttons across the board? If so, then add an `!important` tag alongside the CSS.